# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rockplug/update_cache.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
Update cache after depmod
Alejandro

--- ./etc/rockplug/pci.functions.orig	2005-02-22 00:04:50.863719728 -0300
+++ ./etc/rockplug/pci.functions	2005-02-22 00:05:19.519363408 -0300
@@ -55,7 +55,7 @@
 
   cache="$RPCACHE-pci-$pci_vendor_id.$pci_device_id.$pci_subsys_vendor_id.$pci_subsys_device_id.cache"
 
-if [ ! -e $cache ] ; then
+if [ ! -e $cache -o $MODULES_DIR/modules.pcimap -nt $cache ] ; then
   while read module vendor device subvendor subdevice \
     class class_mask driver_data ignore flags; do
 
--- ./etc/rockplug/usb.functions.orig	2005-02-22 00:06:04.166576000 -0300
+++ ./etc/rockplug/usb.functions	2005-02-22 00:06:15.489854600 -0300
@@ -100,7 +100,7 @@
 
   cache="$RPCACHE-usb-$usb_idVendor.$usb_idProduct.$usb_bcdDevice.$usb_bDeviceClass.cache"
 
-if [ ! -e $cache ] ; then
+if [ ! -e $cache -o $MODULES_DIR/modules.usbmap -nt $cache ] ; then
   while read module match_flags idVendor idProduct bcdDevice_lo \
     bcdDevice_hi bDeviceClass bDeviceSubClass line; do
     set $line ''
